vividus-engine/src/main/java/org/vividus/resource/TestResourceLoader.java

```diff
@@ -40,12 +40,18 @@ public class TestResourceLoader implements ITestResourceLoader
     private final List<String> resourceLoadParameters;
     private final ResourcePatternResolver resourcePatternResolver;
 
-    public TestResourceLoader(IPropertyParser propertyParser, ResourcePatternResolver resourcePatternResolver)
+    private final boolean enableResourceLoaderConfiguration;
+
+    public TestResourceLoader(IPropertyParser propertyParser, ResourcePatternResolver resourcePatternResolver,
+            boolean enableResourceLoaderConfiguration)
     {
-        resourceLoadParameters = propertyParser.getPropertyValuesByPrefix(VARIABLES_PROPERTY_PREFIX).values().stream()
+        resourceLoadParameters = enableResourceLoaderConfiguration
```

Comment:
also it makes to log some warning, that this feature is deprecated and will be removed in 0.7.0